resource_transip_vps.go

```diff
@@ -153,24 +156,24 @@ func resourceVpsCreate(d *schema.ResourceData, m interface{}) error {
 
 	}
 
-	d.SetId(name)
-
 	d.Set("install_text", InstallText)
+	setVpsID(d, m)
```

Comment:
Please inline this code, also ignoring return value.

Reply:
I've moved it to be inline, but I'm not too sure about the return value of `resourceVpsCreate` now, could you check if this is right?